pom.xml

```diff
@@ -106,6 +109,24 @@
 			<artifactId>netty-resolver-dns-native-macos</artifactId>
```

Comment:
why do you need this dependency?

Reply:
I got this error from idea:
"Unable to load io.netty.resolver.dns.macos.MacOSDnsServerAddressStreamProvider, fallback to system defaults. This may result in incorrect DNS resolutions on MacOS. Check whether you have a dependency on 'io.netty:netty-resolver-dns-native-macos"

Reply:
This dependency is removed